pipelines/README.md

```diff
@@ -129,20 +129,29 @@ Parameters:
     `given=Susan` criteria. Default: `Patient,Encounter,Observation`
 -   `batchSize` - The number of resources to fetch in each API call. Default:
     `100`
--   `openmrsFhirBaseEndpoint` - The OpenMRS server base path for its FHIR API
-    endpoints. Using all default values, you would find Patient resources at
-    `http://localhost:8099/openmrs/ws/fhir2/R4/Patient`. This generally should
-    not be changed. Default: `/ws/fhir2/R4`
+    
+### Batch mode using FHIR Search for HAPI source
 
-### Batch mode using JDBC
+To start Batch Mode using FHIR Search, run:
+
+```shell
+$ java -cp batch/target/fhir-batch-etl-bundled-0.1.0-SNAPSHOT.jar \
+    org.openmrs.analytics.FhirEtl \
+    --fhirServerUrl=http://localhost:8098/fhir \
+    --fhirServerUserName=hapi --fhirServerPassword=hapi \
```

Comment:
Are these the default username/password that comes with HAPI official docker image?
Please also consider documenting how you start HAPI. This can go into a separate PR just for documentation changes in which we also update OpenMRS related notes at the top of this page.

Reply:
The official HAPI docker image does not provide a default username/password, and I've used this username/password to stay consistent with the rest of our documentation. I will provide documentation changes in a separate PR that addresses how to start HAPI and updates the OpenMRS related notes as well.